spec/console/console_spec.rb

```diff
@@ -32,24 +32,40 @@
     EXPECTED_OUTPUT = "| X |   | O | "
     @game.board.check_tile(1, 0, @player1)
     @game.board.check_tile(1, 2, @player2)
-    @console.print_row(@game.board, 1)
+    @console.send(:print_row, @game.board, 1)
```

Comment:
send is a method (on Object?) that you should not override.

Reply:
I haven't overridden it: I'm using it to test a private method.

Reply:
By defining the method with the same name in your class, you have overridden it.